### PR TITLE
Add chart config for life expectancy.

### PR DESCRIPTION
**Pull request by tjann**
Local screenshot: 
image

I noticed current staging/prod does not have years for median age so I did the same: 
image

Question: I had some changes to static/package.*json from just running `./run_npm.sh`. Is the correct thing to do: do not commit these changes unless you know what you are doing?

### CODE REVIEW

**Comment by tjann** [comment]
Thanks for the idea @beets ! Here are some screenshots to show the update:

Chart gone from demo
image

Chart now in health
image

How the charts look
image


Out of scope of this PR: exact chart placements (unless someone wants to tell me where and how to update). I also found and filed #474

File: server/chart_config.json [hunk 1]
```diff
@@ -160,6 +159,12 @@
       "denominator": "Count_Household"
     }
   },
+  {
+    "category": ["Health", "Life Expectancy"],
+    "title": "Life Expectancy",
+    "statsVars": ["LifeExpectancy_Person"],
+    "isOverview": true
```
Comment (beets) [inline server/chart_config.json:166]: this is somewhat out of date now. could you please copy over relatedChart as well?

File: server/webdriver_tests/place_explorer_test.py [hunk 2]
```diff
@@ -91,7 +91,7 @@ def test_demographics_link(self):
         chart_title = age_across_places_chart.find_element_by_tag_name(
             "h4").text
         self.assertEqual(
-            "Population by Gender across states near California(2018)",
+            "Median Age by Gender across states near California(2018)",
```
Comment (beets) [inline server/webdriver_tests/place_explorer_test.py:94]: this will have to be updated post-merge from upstream/master as well

**Review by beets — CHANGES_REQUESTED** [review]
thanks - lg in general though there's been a ton of adjacent changes recently

File: server/chart_config.json (hunk 1 above)
Comment (beets) [reply server/chart_config.json:176]: please copy over the relatedChart object from the existing config. and post-merge from upstream, we no longer require the second item in the category array.

**Review by beets — COMMENTED** [review]
don't forget to run ./run_test.py -f to fix the lint errors

File: server/chart_config.json (hunk 1 above)
Comment (tjann) [reply server/chart_config.json]: done

File: server/webdriver_tests/place_explorer_test.py (hunk 2 above)
Comment (tjann) [reply server/webdriver_tests/place_explorer_test.py]: done

**Review by tjann — COMMENTED** [review]
Looks ready to me

**Review by beets — APPROVED** [review]

**Pull request merged**